events/events.go

```diff
@@ -127,9 +127,12 @@ func reconcileResponderEvent(tx *gorm.DB, event api.Event) error {
 	var externalID string
 	if responder.Properties["responderType"] == "Jira" {
 		externalID, err = responderPkg.NotifyJiraResponder(responder)
-		if err != nil {
-			return err
-		}
+	} else if responder.Properties["responderType"] == "MSPlanner" {
+		externalID, err = responderPkg.NotifyMSPlannerResponder(responder)
+	}
+
```

Comment:
@moshloop Should I use a `getResponderFunc(responderType string) func(responder) (string, error)` or a `switch` here or is if/else fine for now

Reply:
I would incline towards using a switch